source/http_stream.c

```diff
@@ -327,3 +327,22 @@ PyObject *aws_py_http_client_stream_new(PyObject *self, PyObject *args) {
     }
     return NULL;
 }
+
+PyObject *aws_py_http_client_stream_activate(PyObject *self, PyObject *args) {
+    (void)self;
+
+    PyObject *py_stream = NULL;
+    if (!PyArg_ParseTuple(args, "O", &py_stream)) {
+        PyErr_SetString(PyExc_TypeError, "stream object must not be null");
+        Py_RETURN_FALSE;
```

Comment:
Just `return NULL;`

for a few reasons:
- If PyArg_ParseTuple() fails, it's already set a helpful python exception explaining why it failed.
- PyArg_ParseTuple() will accept `None` for "O" (object) types, so this error message isn't even accurate

Reply:
fixed